src/dnssec/sign/denial/nsec3.rs

```diff
@@ -470,6 +486,10 @@ where
         last_nent_stack.push(name.clone());
     }
 
+    let Some(nsec3param_ttl) = nsec3param_ttl else {
+        return Err(SigningError::SoaRecordCouldNotBeDetermined);
```

Comment:
Is this the wrong error here?

Reply:
No, it can only happen if for some reason the SOA wasn't seen.

Reply:
Ah, you are right. It is about the TTL.